doc/engine/Running-the-OpenQuake-Engine-on-Multiple-Nodes.md

```diff
@@ -1,12 +1,12 @@
 ## Initial install
-On all nodes, install Ubuntu and the python-oq package as described in [OpenQuake Engine 1.4 installation](Installing-the-OpenQuake-Engine-1.4.md) or [OpenQuake Engine Master installation](Installing-the-OpenQuake-Engine-Nightly.md).
+On all nodes, install Ubuntu and the python-oq package as described in [OpenQuake Engine installation](Installing-the-OpenQuake-Engine-1.4.md) or [OpenQuake Engine Master installation](Installing-the-OpenQuake-Engine-Nightly.md).
```

Comment:
Are you happy with keeping the release number in the name of the page? I mean ``Installing-the-OpenQuake-Engine-1.4.md`` ? A generic page ``Installing-the-OpenQuake-Engine.md`` seems to be better to me (with different content depending on the release branch).